src/applications/disability-benefits/526EZ/validations.js

```diff
@@ -19,3 +19,10 @@ export function requireOneSelected(
     );
   }
 }
+
+export function isInPast(errors, fieldData) {
+  const fieldDate = new Date(fieldData);
```

Comment:
I think this is subject to some controversy. From [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date):
> Note: parsing of date strings with the Date constructor (and Date.parse, they are equivalent) is strongly discouraged due to browser differences and inconsistencies. Support for RFC 2822 format strings is by convention only. Support for ISO 8601 formats differs in that date-only strings (e.g. "1970-01-01") are treated as UTC, not local.

That said, we should be comparing user-clock UTC (from `new Date(fieldData)` -> `.getTime()`) to user-clock UTC (`Date.now()`), so it _should_ be alright unless the user's clock is way off. :crossed_fingers:

Reply:
That was my thinking. Given that it's probably pretty unlikely that someone fills one of these forms out within hours of discharge, I think this is safe enough. If you think it'd be safer to just `moment` everything, I'm open to that as well. Not sure if we have a convention we follow in other places.

Reply:
I think we generally use `moment` everywhere, but I'd like to get away from that if we can help it--if for nothing else, just because of the size of the library.